packages/injected-script/main/GlobalVariableBridge/index.ts

```diff
@@ -0,0 +1,54 @@
+import { apply } from '../intrinsic'
+import { clone_into, handlePromise, sendEvent } from '../utils'
+import type { InternalEvents } from '../../shared'
+
+const hasListened: Record<string, boolean> = { __proto__: null! }
```

Comment:
```suggestion
const hasListened: Record<string, boolean> = Object.create(null)
```

> Deprecated: This feature is no longer recommended.

from <https://mdn.io/__proto__>

Reply:
No. It's not deprecated. Please don't use Object.create.

Reply:
`__proto__` in the object literal is a formal syntax. The `Object.prototype.__proto__` getter and setter is deprecated.